core/src/main/scala/org/http4s/rho/Result.scala

```diff
@@ -7,79 +7,83 @@ import org.http4s.headers.`Content-Type`
 /** A helper for capturing the result types and status codes from routes */
 sealed case class Result[
 F[_],
-+CONTINUE,
-+SWITCHINGPROTOCOLS,
-+PROCESSING,
-
-+OK,
-+CREATED,
-+ACCEPTED,
-+NONAUTHORITATIVEINFORMATION,
-+NOCONTENT,
-+RESETCONTENT,
-+PARTIALCONTENT,
-+MULTISTATUS,
-+ALREADYREPORTED,
-+IMUSED,
-
-+MULTIPLECHOICES,
-+MOVEDPERMANENTLY,
-+FOUND,
-+SEEOTHER,
-+NOTMODIFIED,
-+USEPROXY,
-+TEMPORARYREDIRECT,
-+PERMANENTREDIRECT,
-
-+BADREQUEST,
-+UNAUTHORIZED,
-+PAYMENTREQUIRED,
-+FORBIDDEN,
-+NOTFOUND,
-+METHODNOTALLOWED,
-+NOTACCEPTABLE,
-+PROXYAUTHENTICATIONREQUIRED,
-+REQUESTTIMEOUT,
-+CONFLICT,
-+GONE,
-+LENGTHREQUIRED,
-+PRECONDITIONFAILED,
-+PAYLOADTOOLARGE,
-+URITOOLONG,
-+UNSUPPORTEDMEDIATYPE,
-+RANGENOTSATISFIABLE,
-+EXPECTATIONFAILED,
-+UNPROCESSABLEENTITY,
-+LOCKED,
-+FAILEDDEPENDENCY,
-+UPGRADEREQUIRED,
-+PRECONDITIONREQUIRED,
-+TOOMANYREQUESTS,
-+REQUESTHEADERFIELDSTOOLARGE,
-
-+INTERNALSERVERERROR,
-+NOTIMPLEMENTED,
-+BADGATEWAY,
-+SERVICEUNAVAILABLE,
-+GATEWAYTIMEOUT,
-+HTTPVERSIONNOTSUPPORTED,
-+VARIANTALSONEGOTIATES,
-+INSUFFICIENTSTORAGE,
-+LOOPDETECTED,
-+NOTEXTENDED,
-+NETWORKAUTHENTICATIONREQUIRED
+/* 100 */ +CONTINUE,
+/* 101 */ +SWITCHINGPROTOCOLS,
+/* 102 */ +PROCESSING,
+/* 103 */ +EARLYHINTS,
+
+/* 200 */ +OK,
+/* 201 */ +CREATED,
+/* 202 */ +ACCEPTED,
+/* 203 */ +NONAUTHORITATIVEINFORMATION,
+/* 204 */ +NOCONTENT,
+/* 205 */ +RESETCONTENT,
+/* 206 */ +PARTIALCONTENT,
+/* 207 */ +MULTISTATUS,
+/* 208 */ +ALREADYREPORTED,
+/* 226 */ +IMUSED,
+
+/* 300 */ +MULTIPLECHOICES,
+/* 301 */ +MOVEDPERMANENTLY,
+/* 302 */ +FOUND,
+/* 303 */ +SEEOTHER,
+/* 304 */ +NOTMODIFIED,
+/* 305 */ /* USEPROXY is deprecated https://tools.ietf.org/html/rfc7231#section-6.4.5*/
```

Comment:
Does it hurt anything to continue support for it?  In http4s-core, I've been amazed by the things long presumed dead that people need to do to interface with some other product.  Though it's probably less of a problem serverside than clientside.  I trust your call either way.

Reply:
Its not that too hard to put it back in.  It was already partially removed which is why I just finished the job.  If Http4s supports it, Rho probably should support it too, so I'll put that back in and merge.

Reply:
Seems Http4s has taken this approach: ` // Note: UseProxy is deprecated in RFC7231, so we will not ease its creation here.`  So I'll do the same.  Any user can still just return a response with that status code, it won't get the metadata, and it won't be easy, but they can if they want to.